AsyncSwift/Observed/StampView+Observed.swift

```diff
@@ -8,40 +8,135 @@
+        private let cardSize: CGFloat = UIScreen.main.bounds.width - 32
         
-        private let durationAndDelay: CGFloat = 0.3
+        init() {
+            fetchStampsImages()
+        }
+        
+        private func getEvents() -> [String] {
+            let pwRaw = keyChainManager.getItem(key: keyChainManager.stampKey) as? String
+            guard var convertedStringArray = pwRaw?.convertToStringArray() else { return [] }
+            convertedStringArray.insert("Test", at: 0) // MARK: Test 실제에서는 Next storage 둘다 설정해야함
+            self.events = convertedStringArray.reversed()
+            return events
+        }
         
-        func didTabCard () {
-            cardAnimatonModel.isTapped.toggle()
+        /// Storage에 저장되어 있는 Stamp Image를 가져오는 함수이다.
+        /// - 
+        private func fetchStampsImages(){
+            let events = getEvents()
             
-            if cardAnimatonModel.isTapped { // 카드 회전 연속을 위해서 if문 분리
-                withAnimation(.linear(duration: durationAndDelay)) {
-                    cardAnimatonModel.backDegree = 90
-                }
-                withAnimation(.linear(duration: durationAndDelay).delay(durationAndDelay)) {
-                    cardAnimatonModel.frontDegree = 0
+            events.enumerated().forEach { [weak self] in
+                guard let self = self else { return }
+                let event = $0.element
+                let index = $0.offset
+                Task { @MainActor () -> Void in
+                    guard let cardImageURL = URL(string: "https://raw.githubusercontent.com/Async-Swift/jsonstorage/ver2Test/Images/Stamp/" + event + "/stamp.png")
+                    else { return }
+                    
+                    let cardImageRequest = URLRequest(url: cardImageURL)
+                    let (cardImageData, cardImageResponse) = try await URLSession.shared.data(for: cardImageRequest)
+                    guard let httpsResponse = cardImageResponse as? HTTPURLResponse, httpsResponse.statusCode == 200 else { return }
+                    
+                    guard let cardUIImage = UIImage(data: cardImageData) else { return }
+                    
+                    self.cards[event] = Card(originalPosition: cardInterval * CGFloat(index),
+                                             image: Image(uiImage: cardUIImage))
+                    // 가장 최근의 EventCard가 선택된 상태로 지정하기
+                    if index == 0 {
+                        self.cards[event]?.isSelected = true
+                    }
+                } // Task
+            } // forEach
+        } // fetchStampsImages
+                
+        func openByLink(url: URL) async {
+            // URL Example = https://asyncswift.info?tab=Stamp&event=Conference001
+            // URL Example = https://asyncswift.info?tab=Event
+            
+            guard URLComponents(url: url, resolvingAgainstBaseURL: true)?.host != nil else { return }
```

Comment:
```suggestion
            if URLComponents(url: url, resolvingAgainstBaseURL: true)?.host == nil { return }
```

반대로 생각해도 되지않아서 깔끔하다고 생각되어요 !

Reply:
수정했습니다. 이 같은 경우는 if 문은 한 줄로 쓰는게 일반적일까요??